docs/blocks-storage/store-gateway.md

```diff
@@ -17,13 +17,13 @@ At startup **store-gateways** iterate over the entire storage bucket to discover
 
 While running, store-gateways periodically rescan the storage bucket to discover new blocks (uploaded by the ingesters and [compactor](./compactor.md)) and blocks marked for deletion or fully deleted since the last scan (as a result of compaction). The frequency at which this occurs is configured via `-experimental.tsdb.bucket-store.sync-interval`.
 
-The blocks chunks and the entire index are never fully downloaded by the store-gateway. The index-header is stored to the local disk, in order to avoid to re-download it on subsequent restarts of a store-gateway. For this reason, it's recommended - but not required - to run the store-gateway with a persistent local disk. For example, if you're running the Cortex cluster in Kubernetes, you may use a StatefulSet with a persistent volume claim for the store-gateways.
+The blocks chunks and the entire index are never fully downloaded by the store-gateway. The index-header is stored to the local disk, in order to avoid to re-download it on subsequent restarts of a store-gateway. For this reason, it's recommended - but not required - to run the store-gateway with a persistent disk. For example, if you're running the Cortex cluster in Kubernetes, you may use a StatefulSet with a persistent volume claim for the store-gateways.
 
 _For more information about the index-header, please refer to [Binary index-header documentation](./binary-index-header.md)._
 
 ## Blocks sharding and replication
 
-The store-gateway optionally supports blocks sharding.
+The store-gateway optionally supports blocks sharding. Sharing can be used to horizontally scale blocks in a large cluster without hitting any vertical scalability limit.
```

Comment:
```suggestion
The store-gateway optionally supports blocks sharding. Sharding can be used to horizontally scale blocks in a large cluster without hitting any vertical scalability limit.
```

Do you actually mean a cluster here? I thought this does only apply to large tenants (where large means more than 15M time serieses?) as each tenant has it's own TSDB.

Reply:
I mean cluster. Blocks sharding is used both to scale a large cluster made of many small tenants and a large cluster made of 1 single large tenant.